feat: add `toLocaleString` method to `array/complex64` (#2208)
stdlib-js/stdlib: lib/node_modules/@stdlib/array/complex64/README.md

// returns 6.0
```

<a name="method-to-locale-string"></a>

#### Complex64Array.prototype.toLocaleString( \[locales\[, options]] )

> Returns a string representating the elements of a typed array.

Reviewer: ```suggestion
Serializes an array as a locale-specific string.
```

```javascript
var arr = new Complex64Array( 2 );

arr.set( [ 1.0, 1.0 ], 0 );
arr.set( [ 2.0, 2.0 ], 1 );

var str = arr.toLocaleString();
// returns '1 + 1i,2 + 2i'
```

> By default, the method returns a string representating the typed array element separated by separator `comma(,)`. To specify another locales provide an argument `locales` and `options`.
> 
> -   **locales**: A string with a BCP 47 language tag, or an array of such strings.
> -   **options**: An object with configuration properties.

Reviewer: ```suggestion
The method supports the following arguments:

-   **locales**: a string with a BCP 47 language tag or an array of such strings.
-   **options**: configuration properties.
```
